APISpecification.yaml

```diff
@@ -235,9 +235,54 @@ paths:
                       $ref: '#/components/schemas/PopularSpecies'
         '500':
           $ref: '#/components/responses/500InternalServerError'
+  /api/metadata/genome/{genome_id}/dataset/{dataset_type}/attributes:
```

Comment:
It looks like you can pass release version here as well. Probably as an optional query parameter. We don't need to think about this now; but I am sure we will have to include that later.